### ISSUE TITLE
Not playing new songs

### ISSUE THREAD

**Issue by Garventous** [z-huang/music#1654]
### Checklist

- [X] I am able to reproduce the bug with the [latest debug version](https://github.com/z-huang/InnerTune/actions/workflows/build.yml).
- [X] I've checked that there is no open or closed issue about this bug.
- [X] This issue contains only one bug.
- [X] The title of this issue accurately describes the bug.

### Steps to reproduce the bug

Go to Innertune. Click a random song not from downloads. It says "Timeout" or "Unexpected error"
Redownload it multiple times.
Update my F-Droid.
Still the same.
Fix my wifi.
Same thing.
Reset phone. 
Same thing.
Not working.

### Expected behavior

It should play the video/song normally.

### Actual behavior

Doesn't play and instead shows me error
![Screenshot_20241016-191431](https://github.com/user-attachments/assets/ae4e2c80-ed63-4989-8d50-b7626593639d)

### Screenshots/Screen recordings

Pls help
![Screenshot_20241016-191431](https://github.com/user-attachments/assets/4479be52-bdbe-4bbb-9d7a-773903e634d0)

### Logs

It just doesn't work

### InnerTune version

The latest version

### Android version

Android 13 rn  I think

### Additional information

Please help me

**Comment by xoureldeen** [comment]
same..............
that's made me anger because that's the only music app i use.

**Comment by hewiofhiowa** [comment]
If you login from settings>content>login and login to YouTube music it will fix the issue. You just need an account, not the paid version.

**Comment by xoureldeen** [comment]
> If you login from settings>content>login and login to YouTube music it will fix the issue. You just need an account, not the paid version.

![Screenshot_2024-10-17-04-55-48-195_com.zionhuang.music.jpg](https://github.com/user-attachments/assets/414f342a-f93f-4615-a410-cab05d74cce0)

![Screenshot_2024-10-17-04-55-46-658_com.zionhuang.music.jpg](https://github.com/user-attachments/assets/c1a67941-b7e2-4141-9bcf-4341f5bc9783)

**Comment by krolyk** [comment]
> If you login from settings>content>login and login to YouTube music it will fix the issue. You just need an account, not the paid version.

Thank you. It fixed the issue for me

**Comment by GabubuAvailable** [comment]
I get the same issue as @xoureldeen 
I try to login but it doesn't say anything. It does the same thing as on his screenshots.

**Comment by xoureldeen** [comment]
> I get the same issue as @xoureldeen I try to login but it doesn't say anything. It does the same thing as on his screenshots.

no one will help, let's download yt vance g mod 🤷🏻

**Comment by rayn000** [comment]
I'm also facing the same. Few days back I was facing the exact problem as the original poster mention. But right now it's telling to sign in. But here's the thing, music sometimes plays without sign in (not that I've signed in, yet).

Hope this is something dev can fix. I don't really like to sign in and don't like look for alternative, because after searching the web, this is the app I landed on. sigh~~

![screenshot](https://github.com/user-attachments/assets/b89fb1bb-c281-409d-ab7d-f66033d01384)
